app/routes/invitations.rb

```diff
@@ -3,11 +3,11 @@ module Routes
     class Invitations < Core
       post '/teams/:slug/invitations' do |slug|
         please_login
+        valid_invitee = params[:usernames].delete(current_user.username)
         only_for_team_managers(slug, "You are not allowed to add team members.") do |team|
-          team.invite_with_usernames(params[:usernames], current_user)
+          team.invite_with_usernames(valid_invitee, current_user)
```

Comment:
It might be worth removing the current user from the list of usernames to invite at the `Team` object level as I would think that's something all team creation should do regardless of route. Just a nit.